app/_layouts/extension.html

```diff
@@ -217,10 +217,10 @@
 
       {% if page.enterprise %}
         <div class="alert alert-warning">
-          <strong>Kong Enterprise Edition (EE) only</strong>: This plugin is only available with a <a href="https://konghq.com/kong-enterprise-edition">Kong EE subscription</a>.
+          <strong>Kong Enterprise (EE) only</strong>: This plugin is only available with a <a href="https://konghq.com/kong-enterprise-edition">Kong Enterprise subscription</a>.
```

Comment:
Still says EE

I'm going to stop reviewing here - please address the previous comments through the PR, and I'll be happy to re-review.